include/constants.h

```diff
@@ -46,4 +48,9 @@ constexpr double max_font_size = 10.0;
 // area to be considered "too small"
 constexpr double small_area_threshold_frac = 2e-5;
 
+constexpr double xi_sq = 4.0;
+
+// Grid cell width/height, as multiple of grid size of 1, when plotting heatmap
```

Comment:
Would the following description be clearer? "Distance between plotted grid lines on heatmap, measured in units of the coordinate system in which the inset frame has a width of lx and a height of ly"